Makefile

```diff
@@ -123,3 +123,13 @@ clone-samples:
 		tar -xJvf samples/linux/clean/$$file -C samples/linux/clean; \
 	done
 	tar -xJvf samples/macOS/clean/bincapz.xz -C samples/macOS/clean
+
+CRANE_VERSION=v0.20.2
+.PHONY: install-crane
```

Comment:
Phony targets aren't cached. Instead of a phony target, can you make this a filename-based make target (including the version)?

Reply:
Updated in [`ef6376b` (#438)](https://github.com/chainguard-dev/bincapz/pull/438/commits/ef6376b71e5c4be65ca05a8525231cc2c4fe6f2e). I fixed the OCI test refresh to pull from the `.xz` archive as well (and cleaned up the path to remove `../pkg/action/`).